tracer/build/_build/gitlab.bat

```diff
@@ -6,11 +6,6 @@ SET VSToolsPath=%VSTUDIO_ROOT%\MSBuild\Microsoft\Visual Studio\v16.0
 
 cd c:\mnt\
 
-REM Temporarily fixes to allow building the profiler code from the dd-continuous-profiler-dotnet repo
```

Comment:
This looks good to me, but I'd like to do a GitLab run before approving. Once you've verified that, feel free to resolve this comment

Reply:
I also remove the profiler repo cloning in the `gitlab-ci.yml` file